src/saiga/vision/util/Features.h

```diff
@@ -6,32 +6,90 @@
 
 #pragma once
 
+#include <opencv2/core/types.hpp>
 #include "saiga/vision/VisionIncludes.h"
 
-
 namespace Saiga
 {
 /**
  * Identical to OpenCV's Keypoint struct except it uses
  * Eigen Points and no 'class_id' member.
  * @brief The Keypoint struct
  */
-template <typename T = float>
-struct KeyPoint
+struct Point
```

Comment:
Please remove this struct and use Vec2 instead.
- Some keypoints require sub-pixel accuracy
- Not compatible with other vision functions. (for example projecting a point to the image)